spec/automate_liveness_agent/config_spec.rb

```diff
@@ -125,7 +126,7 @@
 
         let(:expected_message) do
           "Config file '#{config_path}' is missing mandatory setting(s): " <<
-            BASE_CONFIG_DATA.keys.map { |k| "\"#{k}\"" }.join(',')
+            BASE_CONFIG_DATA.keys.map { |k| "\"#{k}\"" }.join(",")
```

Comment:
Yeah, I agree that having two standards that disagree about something as core as what type of quotes to use for strings is annoying. However, I think it's probably good to stick to our own style-guidelines, even when a repo contains cookbooks and applications.